python/GangaDirac/test/Unit/Files/TestDiracFile.py

```diff
@@ -130,21 +130,21 @@ def test_get(df):
 
     df.lfn = 'lfn'
     with patch('GangaDirac.Lib.Files.DiracFile.execute', return_value={'OK': True, 'Value': {'Successful': {'%s' % df.lfn: True}}}) as execute:
-        assert df.get() is None
+        assert df.get() is True
```

Comment:
Ok, I guess it's best in a test to avoid accidentally testing truthyness instead of `True` by accident. It's fine as-is then.